doc/source/user_guide/io.rst

```diff
@@ -4717,6 +4717,7 @@ Several caveats.
 * The ``pyarrow`` engine preserves the ``ordered`` flag of categorical dtypes with string types. ``fastparquet`` does not preserve the ``ordered`` flag.
 * Non supported types include ``Period`` and actual Python object types. These will raise a helpful error message
   on an attempt at serialization.
+* The ``pyarrow`` engine preserves extension data types such as the nullable integer and string data type (requiring pyarrow >= 1.0.0).
```

Comment:
Does this only preserve integer and string extension types or does it preserve *all*? I assume the former but somewhat unclear in note

Reply:
> Does this only preserve integer and string extension types or does it preserve all? I assume the former but somewhat unclear in note

It does preserve it for all dtypes that have implemented the needed protocol methods (`__arrow_array__`, `__from_arrow__`), which right now is integer and string for pandas. And potentially any external EA as well, in case those support those protocols. 

Tried to clarify this in the docs.